src/app/wishlists/wishlists.ts

```diff
@@ -38,14 +38,14 @@ let inventoryRolls: { [key: string]: InventoryWishListRoll } = {};
 export function getInventoryWishListRolls(
   allItems: DimItem[],
   rollsByHash: { [itemHash: number]: WishListRoll[] }
-): { [key: string]: InventoryWishListRoll } {
+): { [key: string]: InventoryWishListRoll } | undefined {
```

Comment:
now getInventoryWishListRolls can return undefined instead of checking is enabled

Reply:
But why return `undefined` instead of `emptyObject`? You could get rid of so many `?`s.